test/game_tests.cpp

```diff
@@ -111,8 +113,8 @@ SCENARIO("Game::simulate") {
             {1, 1},
             {2, 1}}; // attacker_id 1 and 2 both targetting defender_id 1
 
-    Game first_turn_state =
-        game.simulate(first_turn_player_set_targets, initial_spawn_positions);
+    Game first_turn_state = game.simulate(turn++, first_turn_player_set_targets,
+                                          initial_spawn_positions, activations);
```

Comment:
Also write tests which actually ensure the actors arent taking damage you can write in game,attacker and defender.